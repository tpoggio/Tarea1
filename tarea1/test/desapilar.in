apilar 15 4.0
esVaciaPila
desapilar
esVaciaPila
apilar 15 4.0
apilar 95 24.0
cima
desapilar
cima
apilar 95 24.0
cima
apilar 85 2.0
apilar 65 3.0
apilar 25 67.0
apilar 35 -12.0
apilar 75 2.5
apilar 45 2.1
apilar 55 -24.0
apilar 85 32.0
apilar 10 10.0
cima
desapilar
cima
Fin