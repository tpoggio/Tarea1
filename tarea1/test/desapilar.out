1>Apilar. 
2>La pila no está vacia. 
3>Desapilar. 
4>La pila está vacia. 
5>Apilar. 
6>Apilar. 
7>(95,24.00)
8>Desapilar. 
9>(15,4.00)
10>Apilar. 
11>(95,24.00)
12>Apilar. 
13>Apilar. 
14>Apilar. 
15>Apilar. 
16>Apilar. 
17>Apilar. 
18>Apilar. 
19>Apilar. 
20>Apilar. 
21>(85,32.00)
22>Desapilar. 
23>(55,-24.00)
24>Fin.
